rosdep/base.yaml

```diff
@@ -1906,16 +1906,7 @@ libgphoto-dev:
   debian: [libgphoto2-2-dev]
   fedora: [libgphoto2-devel]
   gentoo: [media-gfx/gphoto2]
-  ubuntu:
-    lucid: [libgphoto2-dev]
-    precise: [libgphoto2-2-dev]
-    raring: [libgphoto2-2-dev]
-    saucy: [libgphoto2-6-dev]
-    trusty: [libgphoto2-dev]
-    utopic: [libgphoto2-dev]
-    vivid: [libgphoto2-dev]
-    wily: [libgphoto2-dev]
-    xenial: [libgphoto2-dev]
+  ubuntu: [libgphoto2-dev]
```

Comment:
these rules were not all identical, e.g. precise raring and saucy mapped to a different package.
We could use the wildcard syntax to avoid having to list out every new distro but keeping the mapping for the previous distros that had different rules:
```
  ubuntu:
    '*': [libgphoto2-dev]
    precise: [libgphoto2-2-dev]
    raring: [libgphoto2-2-dev]
    saucy: [libgphoto2-6-dev]
```